user/timepix3/module/src/Timepix3Event2StdEventConverter.cc

```diff
@@ -81,8 +81,16 @@ bool Timepix3TrigEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::
 uint64_t Timepix3RawEvent2StdEventConverter::m_syncTime(0);
 uint64_t Timepix3RawEvent2StdEventConverter::m_syncTime_prev(0);
 size_t Timepix3RawEvent2StdEventConverter::m_clearedHeader(0);
+bool Timepix3RawEvent2StdEventConverter::m_first_time(true);
 bool Timepix3RawEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::StandardEventSP d2, eudaq::ConfigurationSPC conf) const{
 
+  // Time difference as criterion for indirect T0 detection (time to previous hit) in MICRO-seconds:
+  uint64_t delta_t_t0 = conf->Get("delta_t_t0", 1e6); // default: 1sec
+  if(m_first_time){
```

Comment:
You have this `m_first_time` **only** to print a message? Then remove it...

Reply:
Here yes, but I'm using the same `first_time` for reading the calibration - see #16. Since it's unavoidable there, I could use it here as well.